frontend/src/pages/patients/PatientsPage.tsx

```diff
@@ -65,8 +74,111 @@ function patientAliasFilterWhereVariables(value: string): PatientAliasWhere[] {
   }
 }
 
-export const PatientsPage: React.FunctionComponent = (props) => {
+function addCDashToCMOId(cmoId: string): string {
+  return cmoId.length === 6 ? `C-${cmoId}` : cmoId;
+}
+
+export default function PatientsPage({ setUserEmail }: { setUserEmail: any }) {
   const params = useParams();
+  const childRef = useRef<any>(null);
+
+  const [phiEnabled, setPhiEnabled] = useState(false);
+  const [patientIdsTriplets, setPatientIdsTriplets] = useState<
+    PatientIdsTriplet[]
+  >([]);
+  const [patientsListColumns, setPatientsListColumns] =
+    useState(PatientsListColumns);
+  const [popupOpen, setPopupOpen] = useState(false);
+  const [showAlertModal, setShowAlertModal] = useState(false);
+
+  useEffect(() => {
+    function handleLogin(event: any) {
+      if (event.origin !== "http://localhost:4001") return;
```

Comment:
all urls should be passed to app in a global configration object.  no hard coding

Reply:
Agreed (see PR description). There's a bit more legwork to make all of this production-ready, which includes removing this hard coding.

> Getting this ready for production will be a separate ticket, so please ignore instances of “localhost:…“” in the code